### PR TITLE
get rid of stopgap retry and replace with deployment serialization stopgap

### PR DESCRIPTION
**Pull request by nuevoalex**

### CODE REVIEW

File: source/libraries/ContractInterfaces.ts [hunk 1]
```diff
@@ -39,23 +39,12 @@
...
-                    }
-                    return;
-                } catch (e) {
-                    console.log(`Retrying tx. Got error: ${e}`);
-                    err = e;
-                    continue;
-                }
+            const transactionHash = await this.connector.ethjsQuery.sendRawTransaction(signedTransaction);
+            const txReceipt = await this.connector.waitForTransactionReceipt(transactionHash, `Waiting on receipt for tx: ${txName}`);
+            if (txReceipt.status != 1) {
```
Comment (MicahZoltu) [inline source/libraries/ContractInterfaces.ts]: Is `txReceipt.status` a number?  I would have expected it to be a `string` or a `BN`.

File: source/libraries/ContractInterfaces.ts [hunk 2]
```diff
@@ -39,23 +39,12 @@
...
+                throw new Error(`Tx ${txName} failed: ${txReceipt}`);
             }
-            throw err;
+            return;
```
Comment (MicahZoltu) [inline source/libraries/ContractInterfaces.ts]: Can delete this line, it is the last line of the function thus `return` is implicit.

File: source/libraries/ContractInterfaces.ts (hunk 1 above)
Comment (MicahZoltu) [reply source/libraries/ContractInterfaces.ts]: Note: this comment and the one below should actually be on the contract generator.

Comment (nuevoalex) [reply source/libraries/ContractInterfaces.ts]: technically it's a `string | number`. In practice this works as expected

**Pull request merged**